mmv1/templates/terraform/examples/shared_reservation_beta.tf.tmpl

```diff
@@ -0,0 +1,59 @@
+resource "google_project" "owner_project" {
+  provider        = google-beta
+  project_id      = "tf-test%{random_suffix}"
+  name            = "tf-test%{random_suffix}"
+  org_id          = "{{index $.TestEnvVars "org_id"}}"
+  billing_account = "{{index $.TestEnvVars "billing_account"}}"
+  deletion_policy = "DELETE"
+}
+
+
+resource "google_project_service" "compute" {
+  provider = google-beta
+  project  = google_project.owner_project.project_id
+  service  = "compute.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project" "guest_project" {
+  provider        = google-beta
+  project_id      = "tf-test-2%{random_suffix}"
+  name            = "tf-test-2%{random_suffix}"
+  org_id          = "{{index $.TestEnvVars "org_id"}}"
+  deletion_policy = "DELETE"
+}
+
+resource "google_organization_policy" "shared_reservation_org_policy" {
+  provider   = google-beta
+  org_id     = "{{index $.TestEnvVars "org_id"}}"
+  constraint = "constraints/compute.sharedReservationsOwnerProjects"
+  list_policy {
+    allow {
+      values = ["projects/${google_project.owner_project.number}"]
+    }
+  }
+}
+
+resource "google_compute_reservation" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+  project  = google_project.owner_project.project_id
+  name = "{{index $.Vars "reservation_name"}}"
+  zone = "us-central1-a"
+
+  specific_reservation {
+    count = 1
+    instance_properties {
+      min_cpu_platform = "Intel Cascade Lake"
+      machine_type     = "n2-standard-2"
+    }
+  }
+  share_settings {
+    share_type = "SPECIFIC_PROJECTS"
+    project_map {
```

Comment:
> Error: Error creating Reservation: googleapi: Error 400: Invalid value for field 'resource': '{  "zone": "projects/tf-test1k6oh4ckdk/global/zones/us-central1-a",  "name": "tf-test-gce-shared-res...'. Only one of projects or project_map can be used in the request, not both., invalid

Reply:
Why are you unable to test it locally? Are you getting a specific error message?

Reply:
Unfortunately, I do not have the necessary permissions to run the acc test with the required configuration.